apteryxd.c

```diff
@@ -1956,7 +1965,15 @@ handle_timestamp (rpc_message msg)
     if ((value = proxy_timestamp (path)) == 0)
     {
         /* Lookup value */
-        value = db_timestamp (path);
+        if (config_tree_has_providers (path) ||
+            call_refreshers (path, true))
+        {
+            value = calculate_timestamp ();
+        }
+        else
+        {
+            value = db_timestamp (path);
```

Comment:
Current behaviour is that if anything is stored in the DB, then that trumps anything provided. I wonder if this should be
```
if call_refreshers
  calculate_timestamp
else if value in db
  db_timestamp
else if has_providers
  calculate_timestamp
```